src/features/embalm/stepContent/hooks/useCreateSarcophagus.ts

```diff
@@ -174,10 +182,17 @@ export function useCreateSarcophagus() {
 
   const uploadAndSetDoubleEncryptedFile = useCallback(async () => {
     try {
-      const payload = await readFileDataAsBase64(file!);
+      const data = await readFileDataAsBase64(file!);
+      const payload = {
+        fileName: file?.name,
+        data,
+      };
```

Comment:
Can we utilize the `Content-Type` tag, instead of needing to include the filename? 

https://docs.bundlr.network/docs/client/transactions

```
const data = fs.readFileSync("data.txt");

const tags = [{name: "Content-Type", value: "text/plain"}];

const transaction = bundlr.createTransaction(data, { tags: tags });
```

If we can't get the tag from the `api.get` method, then this may not work for goerli and we may need the filename.

Reply:
Absolutely we can utilize `Content-Type`, it's included in the data portion of the payload. However I was trying to preserve the name of the file in this case.

Reply:
We could also create a tag for the filename --- thinking it's a little cleaner to just upload the pure file bytes as the payload.

Reply:
@knoll3 -- actually -- thinking the filename shouldn't be used anywhere --- don't want to call attention to any specific sarcophagus, just in case the filename is like "private_key_to_wallet_with_10000000_dollars"

@katien -- do you have an opinion on this?

Reply:
Just to capture our talk on discord here... The file name will be encrypted so we agreed to keep it as is.